helm/templates/ingress.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
The main structure of the skeleton is kept (services names, etc). I agree about not changing too much overall structure, but within reason:  there's no way this chart is deployed without an ingress hence we don't need this.

Having said that, if you want to keep it, it causes no harm for the moment :smile: